playbooks/example-s3-use/group_vars/all

```diff
@@ -0,0 +1,30 @@
+---
+pulp_default_admin_password: password
+pulp_install_object_storage: s3
+pulp_install_plugins:
+  # galaxy-ng: {}
+  # pulp-2to3-migration: {}
+  # pulp-ansible: {}
+  # pulp-certguard: {}
+  # pulp-container: {}
+  # pulp-cookbook: {}
+  # pulp-deb: {}
+  pulp-file: {}
+  # pulp-gem: {}
+  # pulp-maven: {}
+  # pulp-npm: {}
+  # pulp-python: {}
+  # pulp-rpm: {}
+pulp_settings:
+  secret_key: secret
+  content_origin: "{{ pulp_webserver_disable_https | default(false) | ternary('http', 'https') }}://{{ ansible_fqdn }}"
+  aws_access_key_id = 'AKIAIT2Z5TDYPX3ARJBA'
+  aws_secret_access_key = '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'
+  aws_storage_bucket_name = 'pulp3'
+  aws_default_acl = "@none None"
+  s3_use_sigv4 = True
+  aws_s3_signature_version = "s3v4"
+  aws_s3_addressing_style = "path"
+  aws_s3_region_name = "eu-central-1"
+  default_file_storage = 'storages.backends.s3boto3.S3Boto3Storage'
+  media_root = ''
```

Comment:
like these variables for S3 for example, but I don't know how to communicate it well on the docs